testing-common/src/main/java/io/opentelemetry/instrumentation/testing/junit/db/DbConnectionPoolMetricsAssertions.java

```diff
@@ -8,11 +8,18 @@
 import static io.opentelemetry.api.common.AttributeKey.stringKey;
 import static io.opentelemetry.sdk.testing.assertj.OpenTelemetryAssertions.assertThat;
 
+import io.opentelemetry.api.common.AttributeKey;
 import io.opentelemetry.api.common.Attributes;
 import io.opentelemetry.instrumentation.testing.junit.InstrumentationExtension;
+import io.opentelemetry.sdk.metrics.data.MetricData;
+import io.opentelemetry.sdk.testing.assertj.LongSumAssert;
+import io.opentelemetry.sdk.testing.assertj.MetricAssert;
 
 public final class DbConnectionPoolMetricsAssertions {
 
+  public static final AttributeKey<String> POOL_NAME_KEY = stringKey("pool.name");
+  public static final AttributeKey<String> STATE_KEY = stringKey("state");
```

Comment:
```suggestion
  private static final AttributeKey<String> POOL_NAME_KEY = stringKey("pool.name");
  private static final AttributeKey<String> STATE_KEY = stringKey("state");
```

No need to make public since we will get them in the next semconv update

Reply:
That's cool. I'm sure I just used the refactor default.